modules/seqexec/server/src/main/scala/seqexec/server/gmos/GmosNorth.scala

```diff
@@ -30,18 +30,26 @@ final case class GmosNorth[F[_]: Concurrent: Timer: Logger] private (
   c: GmosNorthController[F],
   dhsClient: DhsClient[F],
   nsCmdR: Ref[F, Option[NSObserveCommand]]
-) extends Gmos[F, NorthTypes](c,
+  ) extends Gmos[F, NorthTypes](c,
   new SiteSpecifics[NorthTypes] {
     def extractFilter(config: CleanConfig): Either[ConfigUtilOps.ExtractFailure, NorthTypes#Filter] =
       config.extractInstAs[NorthTypes#Filter](FILTER_PROP)
     def extractDisperser(config: CleanConfig): Either[ConfigUtilOps.ExtractFailure, GmosNorthType.DisperserNorth] =
       config.extractInstAs[NorthTypes#Disperser](DISPERSER_PROP)
     def extractFPU(config: CleanConfig): Either[ConfigUtilOps.ExtractFailure, GmosNorthType.FPUnitNorth] =
       config.extractInstAs[NorthTypes#FPU](FPU_PROP_NAME)
+    def extractCustomFPU(config: CleanConfig): Either[ConfigUtilOps.ExtractFailure, String] =
+      config.extractInstAs[String]("fpuCustomMask")
```

Comment:
Should we put the property name in a constant for consistency with the other ones?